tests/table-operations.spec.ts

```diff
@@ -89,6 +89,14 @@ async function changeCondFormatHighlight(page: Page) {
   await waitForLoadingDone(page);
 }
 
+async function artificialWait(page: Page) {
```

Comment:
Add some comments on why we need this... basically that sometimes the loading indicator appears so quickly playwright misses it.

I'm still looking for a way to see if you can increase the retry frequency for a test to get around this artificialWait (as I assume that's what the problem is).